spec/controllers/errors_controller_spec.rb

```diff
@@ -1,5 +1,3 @@
-require 'rails_helper'
```

Comment:
As far as I could tell, rails_helper and spec_helper served the same purpose, so I moved everything that was in rails_helper but not in spec_helper into spec_helper and got rid of rails_helper.